src/common/common.vcxproj

```diff
@@ -111,7 +115,9 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
     </Link>
     <Lib>
-      <AdditionalLibraryDirectories>Shlwapi.lib;</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories>
+      </AdditionalLibraryDirectories>
+      <AdditionalDependencies>shlwapi.lib;shcore.lib</AdditionalDependencies>

```

Comment:
`AdditionalLibraryDirectories` would be an additional folder that can be used to search for missing lib files, while `AdditionalDependencies` is an actual set of lib files which it is dependent on. So the shlwapi.lib was supposed to be in `AdditionalDependencies` but it looks like it might have been misplaced in an older PR. Another thing that was missing was shcore.lib since there is a DPI related method in common which required that. These usually don't throw errors on build unless the actual dll project which is using `common` was missing that reference. Which is why if you check FZ and even the KBM project prior to this change, it was referencing `shcore.lib` even though the dependency was actually in common and not in KBM. So moving the dependency there fixes that without having to manually add shcore.lib to any future projects which would use DPI methods from `common`.